controllers/sensor/resource.go

```diff
@@ -66,6 +67,18 @@ func Reconcile(client client.Client, eventBus *eventbusv1alpha1.EventBus, args *
 		return fmt.Errorf("eventbus not ready")
 	}
 
+	if args.Sensor.Spec.LiveReload {
+
+		sensorConfigmMapName := fmt.Sprintf("sensor-cm-%s", args.Sensor.Name)
+		serializedBytes, err := yaml.Marshal(args.Sensor)
+		if err != nil {
+			//TODO: add a log error here
+			return err
+		}
+		sensorContent := map[string]string{"sensor.yaml": string(serializedBytes)}
+		updateOrCreateConfigMap(ctx, client, sensorConfigmMapName, sensor.Namespace, sensorContent)
```

Comment:
I'm wondering if we should call this `buildConfigMap` and return the cm object in order to match the `buildDeployment` pattern?